mcs/class/System.ServiceModel/System.ServiceModel.Description/MetadataExchangeClient.cs

```diff
@@ -88,7 +89,17 @@ public MetadataExchangeClient (Uri address, MetadataExchangeClientMode mode)
 		[MonoTODO]
 		public int MaximumResolvedReferences { get; set; }
 
-		public TimeSpan OperationTimeout { get; set; }
+		public TimeSpan OperationTimeout {
+			get {
+				if (!this.operation_timeout.HasValue) {
+					this.operation_timeout = TimeSpan.FromMinutes (1);
```

Comment:
[MSDN](https://msdn.microsoft.com/en-us/library/system.servicemodel.description.metadataexchangeclient.operationtimeout(v=vs.100).aspx) lists the default as 1 minute on .NET